templates/postgresql.conf.j2

```diff
@@ -0,0 +1,8 @@
+#------------------------------------------------------------------------------
+# PostgreSQL configuration file managed by ansible
+# CUSTOMIZED (ANSIBLE) OPTIONS
+#------------------------------------------------------------------------------
```

Comment:
you shouldn't have this header here - instead you should have this:
`{{ ansible_managed | comment }}`
The user will typically provide their own `ansible_managed` value, or else will use the default value provided by Ansible.  https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-managed-str